images/ad-server/Containerfile

```diff
@@ -1,35 +1,52 @@
-FROM quay.io/samba.org/sambacc:latest AS builder
-ARG SAMBACC_VER=master
-ARG SAMBACC_REPO=https://github.com/samba-in-kubernetes/sambacc
-
-# The SAMBACC_VER value controls what version of sambacc we'll be building.
-# This can be a branch name, a tag, or a version hash.  When building tagged
-# versions of samba-containers we should use a sambacc tag.
-# SAMBACC_DISTNAME controls where the script will output a built wheel.
-RUN SAMBACC_DISTNAME=latest \
-    /usr/local/bin/build.sh ${SAMBACC_VER} ${SAMBACC_REPO}
-
-FROM registry.fedoraproject.org/fedora:36
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
 ARG INSTALL_PACKAGES_FROM=default
 ARG SAMBA_VERSION_SUFFIX=""
 ARG SAMBA_SPECIFICS=
 ARG INSTALL_CUSTOM_REPO=
 
 MAINTAINER John Mulligan <jmulligan@redhat.com>
 
-COPY install-packages.sh /usr/local/bin/install-packages.sh
-RUN /usr/local/bin/install-packages.sh \
-    "${INSTALL_PACKAGES_FROM}" \
-    "${SAMBA_VERSION_SUFFIX}" \
-    "${INSTALL_CUSTOM_REPO}"
+# openSUSE Build Service specific build tags and labels:
+# Defines the tag for OBS and build script builds:
+#!BuildTag: opensuse/samba-ad-server:latest
+#!BuildTag: opensuse/samba-ad-server:%%MINOR%%
+#!BuildTag: opensuse/samba-ad-server:%%PKG_VERSION%%
+#!BuildTag: opensuse/samba-ad-server:%%PKG_VERSION%%-%RELEASE%
+# labelprefix=org.opensuse.samba-ad-server
+LABEL org.opencontainers.image.title="Samba ADDC container"
+LABEL org.opencontainers.image.description="Samba ADDC container"
+LABEL org.opencontainers.image.created="%BUILDTIME%"
+LABEL org.opencontainers.image.version="%%PKG_VERSION%%-%RELEASE%"
+LABEL org.opencontainers.image.vendor="SINK Project"
+LABEL org.openbuildservice.disturl="%DISTURL%"
+LABEL org.opensuse.reference="registry.opensuse.org/opensuse/samba-ad-server:%%PKG_VERSION%%-%RELEASE%"
```

Comment:
However, this annotation is OpenSUSE specific and should be only in OpenSUSE specific  container file (something like `Containerfile.opensuse`).